iotedgedev/modules.py

```diff
@@ -15,6 +14,53 @@ def __init__(self, envvars, utility, output, dock):
         self.dock = dock
         self.dock.init_registry()
 
+    def create(self, name, template):
+        self.output.header("CREATING MODULE")
+
+        repo = "{0}/{1}".format(self.envvars.CONTAINER_REGISTRY_SERVER, name.lower())
```

Comment:
Sanitize name to remove invalid dotnetnamespace chars, like "-"

Reply:
Added input check. Thanks.